### DISCUSSION TITLE
Is it safe to update to 11.2?

### DISCUSSION THREAD

**Discussion by MarkMorrow** [simprecicchiani/ThinkPad-T460s-macOS-OpenCore#92] [Q&A] [answered]
Waiting for advice on if this is the proper way to proceed, and if there are any extra steps or updates required when going from 11.1 to 11.2.

![Screen Shot 2021-02-02 at 2 31 55 PM](https://user-images.githubusercontent.com/6678604/106652521-928fd700-6563-11eb-9b6c-3dd9eb8ced10.png)

**Comment by MarkMorrow** [comment, marked as answer]
Installed and still boots.  Nothing broken that I have noticed, but will need a bit more time to be sure.  One thing to note is that it wants to enable FileVault during the first (post-install) login; I unchecked it for it seemed risky..  I'll report back if I notice any issues now that I've upgraded.

![Screen Shot 2021-02-03 at 11 45 10 AM](https://user-images.githubusercontent.com/6678604/106779735-4ac98800-6615-11eb-9273-656bf9cc7efa.png)

![Screen Shot 2021-02-03 at 11 36 13 AM](https://user-images.githubusercontent.com/6678604/106779764-50bf6900-6615-11eb-8996-d773be7a5b7a.png)

![Screen Shot 2021-02-03 at 11 36 51 AM](https://user-images.githubusercontent.com/6678604/106779785-54eb8680-6615-11eb-9c7c-0bc9710a2c0c.png)

**Reply by 559911** [reply]
thanks for your advise now I’m updating 🍎

**Reply by simprecicchiani** [reply]
Thank you for testing it 🙏.

**Comment by cdpidan** [comment]
![image](https://user-images.githubusercontent.com/8141453/106993129-ffe67680-67b4-11eb-843a-1e60aa07dc0a.png)

**Reply by 559911** [reply]
哈哈哈，狠人

**Reply by MarkMorrow** [reply]
Yes; @cdpidan is blazing the trail for the rest of us.  I think I will wait on that one until the "Beta" is removed.  :)
